manifests/version.pp

```diff
@@ -11,8 +11,8 @@
       if $scl_httpd_version {
         $default = $scl_httpd_version
       }
-      elsif ($::operatingsystem == 'Amazon') {
-        $default = '2.2'
+      elsif ($::operatingsystem == 'Amazon' and $::operatingsystemmajrelease == '2') {
```

Comment:
Is Amazon still considered as part of the Red Hat OS family? Do you have recent facts you can share somewhere? I've never used Amazon Linux.

Reply:
hi @ekohl 
Yep. Still a part of the RedHat
```
facter operatingsystem operatingsystemmajrelease osfamily
operatingsystem => Amazon
operatingsystemmajrelease => 2
osfamily => RedHat
```